manifests/kube-apiserver.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: kube-apiserver
+  namespace: {{ .KubeAPIServerConfig.Namespace }}
+  labels:
+    tier: control-plane
+    k8s-app: kube-apiserver
+    openshift.io/control-plane: true
+    openshift.io/component: api
+  annotations:
+    openshift.io/run-level: 0
```

Comment:
does this end up in the pod metadata?

Reply:
no, just daemonset

Reply:
shouldn't it? It's about admission afaik.